tests/playwright/src/utility/operations.ts

```diff
@@ -185,7 +185,7 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
     .poll(async () => await resourcesPage.resourceCardIsVisible(RESOURCE_NAME), { timeout: 10000 })
     .toBeTruthy();
   const podmanResourceCard = new ResourceConnectionCardPage(page, RESOURCE_NAME, machineVisibleName);
-  await playExpect(podmanResourceCard.providerConnections).toBeVisible({ timeout: 10_000 });
+  await playExpect(podmanResourceCard.providerConnections).toBeVisible({ timeout: 10000 });
```

Comment:
I think it's more readable using the grouping feature `10_000` vs `10000`

so I would keep this convention

Reply:
done, changed.